### DISCUSSION TITLE
Upload LivePhotos with immich-go, is it working for you?

### DISCUSSION THREAD

**Discussion by twau** [simulot/immich-go#66] [Q&A] [answered]
Hi

I created a default Immich installation on a new server according to the default docker-compose setup. And imported a newly created Photo Takeout, I noticed that all my iOS LivePhotos is now created as stacks instead of a "LivePhoto" that works on my old Immich setup. 

As seen below, one picture in the stack is the picture, the other broken thumbnail is the video file from the LivePhoto(saves as .qt if downloaded from immich).
![image](https://github.com/simulot/immich-go/assets/536952/ad7deb72-3fd1-4471-9e01-59a4ca9b0c5f)

Can anyone else check if they can upload LivePhotos with latest immich & immich-go? So I can rule out if the issue is on my end before creating an issue, not sure if the issue is in immich-go or immich itself.

**Comment by simulot** [comment]
I need some samples of those files to improve the program. Could you share some of yours?

**Reply by twau** [reply]
Here is sample

EDIT: Removed sample, now fixed in #67. Thanks!

**Comment by simulot** [comment, marked as answer]
Thanks, i'll check that
